hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/ha/OMFailoverProxyProvider.java

```diff
@@ -141,15 +141,14 @@ protected void loadOMClientConfigs(ConfigurationSource config, String omSvcId)
 
         if (omProxyInfo.getAddress() != null) {
 
-          ProxyInfo<OzoneManagerProtocolPB> proxyInfo =
-              new ProxyInfo(null, omProxyInfo.toString());
 
           // For a non-HA OM setup, nodeId might be null. If so, we assign it
           // a dummy value
           if (nodeId == null) {
             nodeId = OzoneConsts.OM_NODE_ID_DUMMY;
           }
-          omProxies.put(nodeId, proxyInfo);
+          // ProxyInfo will be set during first time call to server.
```

Comment:
This should be 
`ProxyInfo will not be set`?

Reply:
It will be set when we make a first-time call to the server.
getProxy() is called when first-time submitRequest is called.

Reply:
Got it, thatnks!